Assignment1/Assignment1.py

```diff
@@ -0,0 +1,112 @@
+import sys
+
+import pandas
+import plotly.express as px
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer
+
+
+def main():
+    # Load the whole csv into iris dataframe
+    iris = pandas.read_csv("iris.data", header=None)
+
+    # add column names
+    iris.columns = [
+        "Sepal_Length",
+        "Sepal_Width",
+        "Petal_Length",
+        "Petal_Width",
+        "Type",
+    ]
+
+    # Use pandas to return descriptive stats
+
+    print("\nDescriptive Statistics through Pandas\n", iris.describe())
+
+    # Make Plots
```

Comment:
I dunno why `plotly` is so fickle, but try forcing it to display all plots a particular way (like in browser). In both pycharm and vscode the plots run but don't prompt display.